core/.js/src/main/scala/io/udash/utils/FileService.scala

```diff
@@ -0,0 +1,45 @@
+package io.udash.utils
+
+import java.io.IOException
+
+import org.scalajs.dom._
+import scalatags.JsDom.all._
+
+import scala.scalajs.js
+import js.JSConverters._
+import js.typedarray._
+
+import scala.concurrent._
```

Comment:
Wildcard imports are a tool, not the default - `import scala.concurrent.{Future, Promise}` should be sufficient. I'd probably reduce the scope of some of those as well (e.g. by moving `import scalatags.JsDom.all._` inside the `save` method).

Reply:
It should be done